### PR TITLE
fix: edge storage

### PR DESCRIPTION
**Pull request by maxwelbm**
- command update bucket, added field `edge_access` 
- params `continuam_token`, `max_object_count`  added to command get objects to pagination
- body response get object

### CODE REVIEW

**Review by PabloDiehl — APPROVED** [review]

File: storage.yaml [hunk 1]
```diff
@@ -491,6 +491,12 @@ components:
           nullable: true
           format: uri
           example: http://api.example.org/accounts/?page=2
+        continuation_token:
```
Comment (PatrickMenoti) [inline storage.yaml:494]: Do we need to inform all of these values??

Comment (maxwelbm) [reply storage.yaml:494]: Are you talking about the previous fields?

Comment (PatrickMenoti) [reply storage.yaml:494]: I mean : type: string
          nullable: true
          maxLength: 200
          minLength: 10
          pattern: '.*'

Comment (maxwelbm) [reply storage.yaml:494]: lint required 😄

Comment (PatrickMenoti) [reply storage.yaml:494]: :this-is-fine-fire:

**Review by PatrickMenoti — DISMISSED** [review]

**Review by PabloDiehl — DISMISSED** [review]

**Review by PabloDiehl — APPROVED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Review by alisson-marcelo-azion — DISMISSED** [review]
This file should be auto generated. Is it just a copy from https://github.com/aziontech/storage-api/blob/dev/openapi-schema.yml?

**Review by helio-neto — APPROVED** [review]

**Pull request merged**